gptme/skills/examples/python-repl-skill.md

```diff
@@ -0,0 +1,99 @@
+---
+type: skill
+status: active
+match:
+  keywords: [python, repl, interactive]
+  tools: [ipython]
+scripts:
+  - python_helpers.py
+dependencies:
+  - ipython
+  - numpy
```

Comment:
I feel like this should be a requirements.txt or something. We don't want skills to be specific to a particular programming language.